example-apps/babel/src/resources/recipe.js

```diff
@@ -114,36 +114,36 @@ const Recipe = {
           key: 'directions',
           required: true,
           type: 'text',
-          helpText: 'Explain how should one make the recipe, step by step.'
```

Comment:
I'm not exactly sure how these formatting changes got in?

Reply:
This is what `yarn prettier --single-quote --write "**/*.js"` as when run by `lint-staged` gets me. Perhaps these files were previously not run through prettier?